include/faker-cxx/Internet.h

```diff
@@ -132,5 +133,47 @@ class Internet
      * @endcode
      */
     static unsigned httpStatusCode(std::optional<HttpResponseType> responseType = std::nullopt);
+
+    /**
+     * @brief Determines which address class user wants to generate in the ipv4 method.
+     * 
+     */
+    enum class IPv4Class
+    {
+        classA,
```

Comment:
consider replacing it with enum values A, B, C (with upper case) as we have word "class" in enum name so it is a duplicate

Reply:
Done